### ISSUE TITLE
How to package the program? This will pop up every time I package

### ISSUE THREAD

**Issue by lava4-LvLOG** [ShadowMario/FNF-PsychEngine#11780]
### Describe your problem here.

When I using the "lime test windows" it says:
```**
source/FunkinLua.hx:479: characters 10-15 : Class<llua.LuaL> has no field error
source/FunkinLua.hx:497: characters 10-15 : Class<llua.LuaL> has no field error
source/FunkinLua.hx:503: characters 10-15 : Class<llua.LuaL> has no field error
source/FunkinLua.hx:553: characters 10-15 : Class<llua.LuaL> has no field error
source/FunkinLua.hx:559: characters 10-15 : Class<llua.LuaL> has no field error
source/FunkinLua.hx:599: lines 599-603 : Int should be Bool
source/FunkinLua.hx:599: lines 599-603 : ... For function argument 'f'
```

so what's wrong with this...?

### Are you modding a build from source or with Lua?

Source

### What is your build target?

Windows x64

### Did you edit anything in this build? If so, mention or summarize your changes.

Yes, I downloaded the "MainMenuState.hx" for new menu UI, And I also change the "Project.xml" for changing title.

**Comment by lava4-LvLOG** [comment]
I'm in a hurry. Please reply me if you can as fast as possible 😭

**Comment by TheKitBoi** [comment]
use this
```
haxelib remove linc_luajit
haxelib git linc_luajit https://github.com/nebulazorua/linc_luajit
```

**Comment by lava4-LvLOG** [comment]
> use this
> 
> ```
> haxelib remove linc_luajit
> haxelib git linc_luajit https://github.com/nebulazorua/linc_luajit
> ```

Thanks dude but...
source/Achievements.hx:132: characters 30-54 : Warning : `FlxCamera.defaultCameras` is deprecated, use `FlxG.cameras.setDefaultDrawTarget` instead
source/MainMenuState.hx:67: characters 3-39 : Warning : `FlxCamera.defaultCameras` is deprecated, use `FlxG.cameras.setDefaultDrawTarget` instead
C:\Users\LvLog\Desktop\Psych\export\release\windows\obj>setlocal enabledelayedexpansion
Warning: Could not find Visual Studio 2017 VsDevCmd
Missing HXCPP_VARS
Error: Could not automatically setup MSVC

**Comment by TheKitBoi** [comment]
install VSC 2019 https://visualstudio.microsoft.com/vs/older-downloads/#visual-studio-2019-and-other-products

and install the right library's
```
MSVC v142 - VS 2019 C++ x64/x86 build tools
Windows SDK (10.0.17763.0)
```

**Comment by lava4-LvLOG** [comment]
> install VSC 2019 https://visualstudio.microsoft.com/vs/older-downloads/#visual-studio-2019-and-other-products
> 
> and install the right library's
> 
> ```
> MSVC v142 - VS 2019 C++ x64/x86 build tools
> Windows SDK (10.0.17763.0)
> ```

Thanks dude, I'm trying

**Comment by lava4-LvLOG** [comment]
> install VSC 2019 https://visualstudio.microsoft.com/vs/older-downloads/#visual-studio-2019-and-other-products
> 
> and install the right library's
> 
> ```
> MSVC v142 - VS 2019 C++ x64/x86 build tools
> Windows SDK (10.0.17763.0)
> ```

Um, I install VSC2019 and install the right library, what should I do next?

**Comment by lava4-LvLOG** [comment]
> install VSC 2019 https://visualstudio.microsoft.com/vs/older-downloads/#visual-studio-2019-and-other-products
> 
> and install the right library's
> 
> ```
> MSVC v142 - VS 2019 C++ x64/x86 build tools
> Windows SDK (10.0.17763.0)
> ```

And this happening...
source/MainMenuState.hx:67: characters 16-36 : Cannot rebind this method : please use 'dynamic' before method declaration
source/Achievements.hx:132: characters 30-33 : Type not found : Flx

**Comment by TWgames2020** [comment]
First, at least look at what he doesn't like there
source/Error script name:line is, as for me, the more important thing that is needed. Well, don't forget about the error that is written in cmd. If possible, leave the screenshots or the code itself. If you haven't changed anything much, then download the Source of Psych engine and then change your files to those in the archive. Well, first save your files somewhere else so that you can add them to a fresh engine later. I did so and it helped me.
Or try to remember after which the error appears and cancel these actions

**Comment by lava4-LvLOG** [comment]
> First, at least look at what he doesn't like there source/Error script name:line is, as for me, the more important thing that is needed. Well, don't forget about the error that is written in cmd. If possible, leave the screenshots or the code itself. If you haven't changed anything much, then download the Source of Psych engine and then change your files to those in the archive. Well, first save your files somewhere else so that you can add them to a fresh engine later. I did so and it helped me. Or try to remember after which the error appears and cancel these actions

Yes, I download the source of Psych Engine change the MainMenuState.hx, But it failed
But I use the original Source Code By ShadowMario(PE 0.6.3), It failed too...

**Comment by ReyRoze** [comment]
Try and re unzip the file again. I had did it and its now compiling

**Comment by ReyRoze** [comment]
unless you already tried it

> Try and re unzip the file again. I had did it and its now compiling
